pipeline/train/configs/opustrainer/student.yml

```diff
@@ -14,6 +14,8 @@ modifiers:
 # boost upper case a little as we see that the models underperform on upper case dataset on evaluation
 - UpperCase: 0.07 # Apply randomly to 7% of sentences
 - TitleCase: 0.05
+# Remove terminal punctuation to teach the model translate text without it
+- RemoveEndPunct: 0.2
```

Comment:
@ZJaume do you think 0.2 probability is sufficient? I see in my de-en sample 77% of data has terminal punctuation, 70% has matching punctuation, and 65% has full stops. So it seems the prob should be relatively high to teach the model not to rely on it.